conf/application.conf

```diff
@@ -189,9 +189,11 @@ urls {
     aftContinueReturn = "http://localhost:8206/manage-pension-scheme-accounting-for-tax/%s/new-return/select-quarter-in-progress"
     aftAmendLink = "http://localhost:8206/manage-pension-scheme-accounting-for-tax/%s/previous-return/amend-select"
     psaFinancialStatementLink = "http://localhost:8206/manage-pension-scheme-accounting-for-tax/view-penalties"
-    paymentsAndChargesLink = "http://localhost:8206/manage-pension-scheme-accounting-for-tax/%s/%s/payments-and-charges"
+    paymentsAndChargesLogicLink = "http://localhost:8206/manage-pension-scheme-accounting-for-tax/%s/past-payments-logic"
```

Comment:
I don't understand why this needs to be an app conf file: can't we just use routes in usual way to get this controller (since it is in the same service)?

Reply:
This gets passed in a partial and rendered in manage

Reply:
Yes, but I can't see any reason why it needs to be in app conf, can still use routes. Anyway, not to worry, this wasn't added as part of this feature branch and it's not a big deal anyway so it can stay as is for now I guess.

Reply:
They don't work if they're used as routes. I have confirmed this while implementing the first AFT partial we implemented. Manage will try to resolve the url at runtime and try to look into it's own routes and not find them there.

Reply:
Ah right, interesting behaviour! Sometimes play routes files are very annoying.. Anyway, fair enough, I've approved PR.